docs/USAGE.md

```diff
@@ -0,0 +1,267 @@
+# Usage
+
+
+- `with`
+  - [`exclude`](#exclude)
+  - [`include`](#include)
+  - [`post-checkout-steps`](#post-checkout-steps)
+  - [`post-install-steps`](#post-install-steps)
+  - [`post-test-steps`](#post-test-steps)
+  - [`runs-on`](#runs-on)
+  - [`strategy-fail-fast`](#strategy-fail-fast)
+  - [`strategy-max-parallel`](#strategy-max-parallel)
+  - [`test-command`](#test-command)
+  - [`upgrade-policy`](#upgrade-policy)
+- `secrets`
+  - [`test-secrets`](#test-secrets)
+
+
+## Parameters (inputs, allowed [via `jobs.<job_id>.with`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idwith))
+
+### `exclude`
+
+A string with matrix combinations to be skipped as a YAML/JSON array of objects with the following keys: `node-version`, `runs-on`.
+
+See also:
+
+- [Github Actions matrix documentation](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix)
+- [`include`](#include)
+- [`runs-on`](#runs-on)
+
+
+#### Example
+
+```yaml
+jobs:
+  test:
+    uses: pkgjs/action/.github/workflows/node-test.yaml@main
```

Comment:
I think that documenting the `node-test.yaml@main` could hurt us when there will be some breaking changes when this GHA will evolve.

Do you think that cutting the `v1` tag would be better?

Adding this workflow:

https://github.com/fastify/github-action-merge-dependabot/blob/main/.github/workflows/release.yml#L25

to this repository would make it even easier to release

Reply:
Yes, I will tag `v1` as soon as everything is ready.

I haven't looked into the details of that workflow, but I've seen it exists - will definitely consider it when the time comes.